benchmark/bench.js

```diff
@@ -1,140 +1,141 @@
-import Conflux from '../write.js'
+import Conflux from '../write';
 
 const {
   WritableStream,
   Response,
   Benchmark,
   Blob,
   JSZip,
-  zip
-} = window
+  zip,
+} = window;
 
-zip.useWebWorkers = false
+zip.useWebWorkers = false;
 
-const uint8 = new Uint8Array(256)
-for (let i = 0; i < uint8.length; i++) uint8[i] = i
+const uint8 = new Uint8Array(256);
+for (let i = 0; i < uint8.length; i += 1) uint8[i] = i;
```

Comment:
do we have to use +1?

Reply:
I can definitely turn off [no-plusplus](https://eslint.org/docs/rules/no-plusplus). It's not a performance thing.
>Because the unary ++ and -- operators are subject to automatic semicolon insertion, differences in whitespace can change semantics of source code.